frontend/src/components/DashboardTable.vue

```diff
@@ -183,43 +184,61 @@ export default {
     redirectToInbox(){
       router.push('/inbox');
     },
+    isThisLoggedInDistrictUser(){
+      return this.userInfo.activeInstituteType === 'DISTRICT';
+    },
     getSchoolsLastUpdateDate() {
-      //The school tile should only show if the userInfo.activeInstituteType == DISTRICT
-      this.loadingTable = true;
-      this.requests = [];
+      if(this.isThisLoggedInDistrictUser()){
+        this.loadingTable = true;
+        this.requests = [];
+        for(const schoolId of this.userInfo.userSchoolIDs) {
```

Comment:
small nitpik, we're trying to transition to be consistent with ID instead of Id. 

Mind using schoolID instead? Be careful though there are some instances where the property names in objects are Id. It's a work in progress to get it all consistent heh.